parm/atm/obs/config/amsua_metop-a.yaml

```diff
@@ -0,0 +1,484 @@
+
+# Calculate Innovation
+- filter: Variable Assignment
+  assignments:
+  - name: DerivedMetaData/Innovation
+    channels: *amsua_metop-a_channels
+    type: float
+    function:
+      name: ObsFunction/Arithmetic
+      channels: *amsua_metop-a_channels
+      options:
+        variables:
+        - name: ObsValue/brightnessTemperature
+          channels: *amsua_metop-a_channels
+        - name: HofX/brightnessTemperature
+          channels: *amsua_metop-a_channels
+        coefs: [1, -1]
+
+# Step 1: Assign initial all-sky observation error
+- filter: Perform Action
+  filter variables:
+  - name: brightnessTemperature 
+    channels: *amsua_metop-a_channels
+  action: 
+    name: assign error
+    error function: 
+      name: DerivedMetaData/InitialObsError
+      channels: *amsua_metop-a_channels
+
+# Step 2: CLW Retrieval Check (observation_based)
+- filter: Bounds Check
+  filter variables:
+  - name: brightnessTemperature
+    channels: 1-6, 15
+  test variables:
+  - name: DerivedMetaData/CLWRetFromObs
+  maxvalue: 999.0
+  actions:
+    - name: set
+      flag: CLWRetrievalReject
+    - name: reject
+
+# Step 3: CLW Retrieval Check (background_based)
+- filter: Bounds Check
+  filter variables:
+  - name: brightnessTemperature
+    channels: 1-6, 15
+  test variables:
+  - name: DerivedMetaData/CLWRetFromBkg
+  maxvalue: 999.0
+  actions:
+    - name: set
+      flag: CLWRetrievalReject
+    - name: reject
+
+# Step 4: Window channel sanity check
+- filter: Bounds Check
+  filter variables:
+  - name: brightnessTemperature
+    channels: 1-6, 15 
+  test variables:
+  - name: DerivedMetaData/Innovation
+    channels: 1, 2, 4-6, 15 
+  maxvalue: 200.0
+  minvalue: -200.0
+  flag all filter variables if any test variable is out of bounds: true
+
+# Step 5: Hydrometeor Check (cloud/precipitation affected chanels)
+- filter: Bounds Check
+  filter variables:
+  - name: brightnessTemperature
+    channels: *amsua_metop-a_channels
+  test variables:
+  - name: ObsFunction/HydrometeorCheckAMSUA
+    channels: *amsua_metop-a_channels
+    options:
+      channels: *amsua_metop-a_channels
+      obserr_clearsky: [ 2.500, 2.200, 2.000, 0.550, 0.300,
+                         0.230, 0.230, 0.250, 0.250, 0.350,
+                         0.400, 0.550, 0.800, 4.000, 3.500]
+      clwret_function:
+        name: DerivedMetaData/CLWRetFromObs
+      obserr_function:
+        name: DerivedMetaData/InitialObsError
+        channels: *amsua_metop-a_channels
+  maxvalue: 0.0
+  actions:
+    - name: set
+      flag: HydrometeorCheckReject
+      ignore: rejected observations
+    - name: reject
+
+# Step 6: Observation error inflation based on topography check
+- filter: Variable Assignment
+  assignments:
+  - name: DerivedMetaData/ObsErrorFactorTopo
+    channels: *amsua_metop-a_channels
+    type: float
+    function:
+      name: ObsFunction/ObsErrorFactorTopoRad
+      channels: *amsua_metop-a_channels
+      options:
+        sensor: *Sensor_ID
```

Comment:
alias `*Sensor_ID` is referenced here and elsewhere, but `&Sensor_ID` is not defined in this yaml file.   

Job gdasatmanlinit failed when processing this yaml with the error

```
yaml.composer.ComposerError: found undefined alias 'Sensor_ID'
  in "/work2/noaa/da/rtreadon/gdas-validation/global-workflow/sorc/gdas.cd/parm/atm/obs/config//amsua_metop-a.yaml", line 312, column 17
```

This comment applies to other amsua yamls in this PR.